### PR TITLE
Chris ogr errors maassluis vlaardingen

### PR DESCRIPTION
**Pull request by Ckerklaan**
Dag Reinout en Arnold,

Heb zojuist aanpassingen gedaan aan de datachecker. Zou een van jullie deze willen controleren? De details staan in de push samenvattingen

### CODE REVIEW

File: tool_quality_checks/fix_import_file.py [hunk 1]
```diff
@@ -248,7 +252,7 @@ def fix_vector_layer(in_layer, layer_name="", epsg=3857):
             break
 
     logger.info("check  - Features count")
-    out_feature_count = out_layer.GetFeatureCount()
+    out_feature_count = out_layer.__len__()
```
Comment (reinout) [inline tool_quality_checks/fix_import_file.py:255]: Dit is normaliter hetzelfde als:

```suggestion
    out_feature_count = len(out_layer)
```

Die dubbele-underscore methodes moet je normaliter niet zelf aanroepen. Meestal zijn het methodes die door standaard python zaken gebruikt worden. `str(iets)` roept achter de schermen effectief `iets.__str__()` aan, bijvoorbeeld. Idem dus met `len()`.

File: tool_quality_checks/pytest.ini [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-[pytest]
-addopts = --cov --flakes --black
-filterwarnings =
-    ignore::DeprecationWarning
\ No newline at end of file
+#[pytest]
+#addopts = --cov --flakes --black
+#filterwarnings =
+#    ignore::DeprecationWarning
```
Comment (reinout) [inline tool_quality_checks/pytest.ini:6]: Is het inderdaad de bedoeling dat dit weg gaat?

File: tool_quality_checks/test/test_copy_ogr_layer.py [hunk 3]
```diff
@@ -142,3 +146,22 @@ def test_get_projection_good(self):
...
+        proj = predict_projection((140205, 140206, 455148, 455149))
+        assert proj.GetAuthorityCode("PROJCS") == "28992"
+
+    def test_predict_projection_4326(self):
+        proj = predict_projection((5.138, 5.139, 52, 52.053))
+        assert proj.GetAuthorityCode("GEOGCS") == "4326"
+
+    def test_predict_projection_28992_not_good(self):
+        proj = predict_projection((56867, 56866, 247873, 247874))
+        assert proj == None
```
Comment (reinout) [inline tool_quality_checks/test/test_copy_ogr_layer.py:164]: Kijk! Hier zijn tests inderdaad makkelijk voor. Dan kan je jezelf makkelijk controleren en later ook nog eens makkelijk corner cases testen.

File: tool_quality_checks/fix_import_file.py [hunk 4]
```diff
@@ -13,6 +13,8 @@
 logger = logging.getLogger(__name__)
 ogr.UseExceptions()
 
+os.environ["SHAPE_ENCODING"] = "ISO-8859-4"
```
[the remark this answers is not in the document]
Comment (Ckerklaan) [reply tool_quality_checks/fix_import_file.py:16]: Ik heb hem in __init__ gezet met een log :)

File: tool_quality_checks/copy_ogr_layer.py [hunk 5]
```diff
@@ -115,6 +130,23 @@ def copy2ogr(in_source, in_name, out_source, out_name, schema="public"):
...
+    """ Return epsg based on y coordinates"""
+    minx, maxx, miny, maxy = bbox
+    if miny > 285000 and miny < 628000:
+        epsg = 28992
+    elif miny > 6500000 and miny < 7200000:
+        epsg = 3857
+    elif miny > 50 and miny < 53:
+        epsg = 4326
+    else:
+        logger.warning("[!] Warning : Projection cannot be predicted by y coordinate")
```
[the remark this answers is not in the document]
Comment (Ckerklaan) [reply tool_quality_checks/copy_ogr_layer.py:143]: Goede suggestie, heb ik toegevoegd!

File: .spyproject/config/backups/codestyle.ini.bak [hunk 6]
```diff
@@ -0,0 +1,8 @@
+[codestyle]
```
[the remark this answers is not in the document]
Comment (Ckerklaan) [reply .spyproject/config/backups/codestyle.ini.bak:1]: Deze heb ik verwijderd :)
